src/Components/Publishing/Display/Canvas/index.tsx

```diff
@@ -42,35 +46,69 @@ export class DisplayCanvas extends React.Component<DisplayCanvasProps> {
     // noop
   }
 
-  render() {
-    const { unit, campaign, article, renderTime } = this.props
+  renderCanvasContent() {
+    const {
+      unit,
+      campaign,
+      article,
+      adUnit,
+      adDimension,
+      renderTime,
+    } = this.props
     const url = unit.link ? get(unit, "link.url", "") : ""
+    const allowedUsers = (sd.HASHTAG_LAB_ADS_ALLOWLIST || "")
+      .split(",")
+      .filter(Boolean)
+    const currentUser = get(sd, "CURRENT_USER.email") || ""
```

Comment:
I added it as null safe check bc if I recall if  what lodash get returns is null it will return null instead of defaulting to the 3rd param, e.g. In `const x = { y: null, z: true }`  `_.get(x, 'y', '')` would return `null` while `x.y || ''` would return `""`

Reply:
so maybe this should be `get(sd, "CURRENT_USER.email", "")` instead?

Reply:
Also Circle failed b/c sd.sd.HASHTAG_LAB_ADS_ALLOWLIST in the test runner so this was my hack to get around that!

Reply:
Also note that we typically use our own implementation of `get`, which is type-safe: https://github.com/artsy/reaction/blob/master/src/Utils/get.ts. (Lots of examples of how its used in the codebase.)